udata/core/dataset/tasks.py

```diff
@@ -157,6 +158,7 @@ def export_csv_for_model(model, dataset):
         csvfile.seek(0)
         # make a resource from this tmp file
         created, resource = store_resource(csvfile, model, dataset)
+        csvfile.close()
```

Comment:
Maybe use the context manager to avoid unclosed files on error or move this into the [`finally` block](https://github.com/opendatateam/udata/blob/1b371426cfe4a126327344775cd4047d9a2bf89a/udata/core/dataset/tasks.py#L167-L168)